k8s/templates/db-migration-job.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: db-migrate
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-delete-policy": hook-succeeded
```

Comment:
for the deletion of the hook, does it mean rolling back the migrations that have been applied?

Reply:
I think it just means we delete the job, so cleanup means the job isn't lying around, and we can create another one with the same name!